apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/TableOperationTask.java

```diff
@@ -24,7 +28,20 @@ protected boolean launchJob() {
         String.format(
             "%s_%s_%s", getType(), getMetadata().getDbName(), getMetadata().getTableName());
     jobId =
-        jobsClient.launch(jobName, getType(), getMetadata().getCreator(), getArgs()).orElse(null);
+        jobsClient
+            .launch(
+                jobName, getType(), getMetadata().getCreator(), getExecutionProperties(), getArgs())
+            .orElse(null);
     return jobId != null;
   }
+
+  protected Map<String, String> getExecutionProperties() {
```

Comment:
It seems like this Map is a generic Map and can contain other properties apart from maintenance.

Reply:
spark properties responsible for maintenance are group under "maintenance." prefix in the table properties, the Jobs API will accept any Spark properties that user provides